challenges/print_level_order/print_level_order.py

```diff
@@ -19,12 +19,11 @@ def recurse(nodelist):
 
         string = ' '.join(printlist)
         all_strings.append(string)
```

Comment:
It is not necessary to declare that `all_strings` is `nonlocal` because all you are doing is appending to the list. If you were reassigning `all_strings` to a new list, then that would be necessary.